### ISSUE TITLE
[Enhancement] ability to add songs from albums or folders to playlist. 

### ISSUE THREAD

**Issue by pokegamer5547** [HardcodeCoder/PulseMusic#104]
**Is the enhancement related to a problem? Please describe.**
when trying to add songs to playlist, it's really gets hard to add songs in a bulk like from an album or folder because there is no way to multi select the songs in the list and adding each song manually takes so much time.

**Describe the solution you'd like**
adding ability to select multiple songs from list to add to playlist can fix up. 

**Describe alternatives you've considered**
giving a option to add songs from folder or album in playlist add options can help too. 

**Additional context**
This can make user experience much better when adding songs to playlist.

**Comment by HardcodeCoder** [comment]
Yes, I agree adding multi select options in albums/library will help. We can also add an option to include all tracks from a given album.

**Comment by pokegamer5547** [comment]
> Yes, I agree adding multi select options in albums/library will help. We can also add an option to include all tracks from a given album.

Yes that's true. That can make things a lot easier for users
